config.json

```diff
@@ -11,7 +11,8 @@
     {
       "platform": "Ecobee 3 Sensors",
       "name": "Ecobee",
-      "exclude_thermostat": false
+      "exclude_thermostat": false,
+      "apiKey":"YOUR API KEY"
```

Comment:
What do you think, about a change to something like this?

```js
"app_key": "Remove this entry if you don't have your Ecobee developer API key."
```

- Adding a longer description will lower the barrier for entry for new users. If we leave it in the `config.json`, people may incorrectly assume that they actually need the API key to use the plugin.

- I think underscore `_` instead of `camelCase` will make the name more consistent with the already existing `exclude_thermostat` option. It's a tiny detail, but it's hard to change it later when people are using it, since such update would break their working setup.